### DISCUSSION TITLE
Try Quasar Framework for cross platform experience?

### DISCUSSION THREAD

**Discussion by iWangJiaxiang** [jellyfin/jellyfin-vue#428] [Questions] [answered]
Hi guys, do you want to try vue-based framework quasar for cross platform experience?

And this is the reason [Why Quasar? | Quasar Framework](https://quasar.dev/introduction-to-quasar)

**Comment by heyhippari** [comment, marked as answer]
(I converted this into a discussion, since it's not really an issue)

We have looked at Quasar before starting the project, as well as most other Vue-based component frameworks. Unfortunately, it didn't meet our needs as far as upstream support goes (See https://www.npmtrends.com/vuetify-vs-quasar-framework)

We settled on Vuetify because it's well supported, does what we need, has nice features we want to take advantage of (Dark mode) and is widely used.

Concerning native apps, the long term plan is for at least Android to migrate to a fully native app. We can't really do native apps on smart TVs, so that only leaves us with iOS, which may become a Swift app at some point if someone is willing to work on it. As a result, I don't thing it's worth it to switch this project (which is already well underway) to another framework only for the promise of cross-platform, when PWA works for now and existing apps depending on the old web client will likely either just depend on the Vue client directly or move towards becoming fully native themselves.

Sorry :/
